project/src/components/offers-list/offers-list.tsx

```diff
@@ -0,0 +1,39 @@
+import OfferCard from '../offer-card/offer-card';
+import Map from '../map/map';
+import Sorting from '../sorting/sorting';
+import { offers } from '../../mocks/offers';
+import { useState } from 'react';
+import { LocationItem } from '../../const';
+
+type OffersListProps = {
+  location: LocationItem,
+  placesCount: number
+}
+
+function OffersList({ location, placesCount }: OffersListProps) {
+  const [activeOfferCard, setActiveOfferCard] = useState('');
```

Comment:
```suggestion
  const [activeOfferCard, setActiveOfferCard] = useState<string>('');
```
всегда типизируй стейты. И лучше сюда